### ISSUE TITLE
Syntax highlighting of detail views

### ISSUE THREAD

**Issue by prurph** [raycast/extensions#4137]
### Description

Both `Detail` and `List.Item.Detail` do not appear to support syntax highlighting of markdown code blocks, (e.g. those starting with ```javascript).

It would be great if this were supported so previews that involve code looked nice.

### Who will benefit from this feature?

Raycast users who interact with code documentation, etc. would benefit from the ability to see a preview in the detail views.

**Comment by ben-greenwood** [comment]
Agreed, would love this for https://www.raycast.com/ben-greenwood/code-stash

**Comment by mathieudutour** [comment]
This should be available in 1.50.0!

**Comment by gobear6212** [comment]
May I ask if there'll be docs on what languages (and what aliases) are supported for syntax highlighting?
